### PR TITLE
simple output message for test

### PR DESCRIPTION
**Pull request by tballmsft**
this works nicely for our rotary encoder

### CODE REVIEW

File: src/testdom/nodes.ts [hunk 1]
```diff
@@ -482,6 +482,9 @@ export class ServiceCommandTest extends ServiceMemberTestNode {
     override updateState() {
         if (this.options?.hasChildren) super.updateState()
     }
+    public setOutput(s: string) {
```
Comment (pelikhan) [inline src/testdom/nodes.ts:485]: why do we need this?

**Comment by pelikhan** [comment]
sync main, https://github.com/microsoft/jacdac-ts/commit/7a3b6e181b9b4c34d2403b630a00d7676df12530

File: src/testdom/nodes.ts (hunk 1 above)
Comment (pelikhan) [reply src/testdom/nodes.ts:485]: i made the output setter public, not needed.

Comment (tballmsft) [reply src/testdom/nodes.ts:485]: it was protected.

Comment (tballmsft) [reply src/testdom/nodes.ts:485]: image

Comment (tballmsft) [reply src/testdom/nodes.ts:485]: you can set test.status, but not test.output

Comment (pelikhan) [reply src/testdom/nodes.ts:485]: You need to merge main, you will have merge issues. output is public in main.

**Pull request closed without merging**